tools/benchmark-cli/src/main/resources/org/logstash/benchmark/cli/cases/app-crashes.cfg

```diff
@@ -0,0 +1,28 @@
+input {
```

Comment:
I don't think we should ship this as a default test case since it requires a running Kafka instance with specific data in it for input and output.

Reply:
Makes sense, will remove.

Reply:
Have removed this file.